From 1511d0e8f43bbff718747c8e4c0cb79892bd2dc1 Mon Sep 17 00:00:00 2001
Message-Id: <1511d0e8f43bbff718747c8e4c0cb79892bd2dc1.1374136911.git.mkletzan@redhat.com>
From: Cole Robinson <crobinso@redhat.com>
Date: Mon, 27 May 2013 18:50:25 -0400
Subject: [RHEL-6.5 python-virtinst PATCH 1/5] details: Fix changing
 cirrus->QXL for active VM (bz 928882)

https://bugzilla.redhat.com/show_bug.cgi?id=980334
(cherry picked from commit 14578fc2c30b4e694db66443e0dca22ba9727c84)

Conflicts:
	virtManager/domain.py -- projects not merged in rhel6, this
and other conflicts will be resolved by cherry-picking to both
projects and requiring same versions in specfile.
---
 virtinst/VirtualVideoDevice.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/virtinst/VirtualVideoDevice.py b/virtinst/VirtualVideoDevice.py
index 138b151..256ebba 100644
--- a/virtinst/VirtualVideoDevice.py
+++ b/virtinst/VirtualVideoDevice.py
@@ -64,6 +64,9 @@ class VirtualVideoDevice(VirtualDevice.VirtualDevice):
         self._vram = val
     vram = _xml_property(get_vram, set_vram,
                          xpath="./model/@vram")
+    ram = _xml_property(lambda o: None, lambda o, v: None,
+                        xpath="./model/@ram")
+

     def get_heads(self):
         return self._heads
-- 
1.8.3.2

